src/Restaurant.js

```diff
@@ -1,28 +1,26 @@
 import React, { useState } from "react";
+import RestaurantButton from "./RestaurantButton";
 
 const Restaurant = () => {
   // array destructuring
   // setOrders is a function that allows me to modify the value of the orders variable
   // 0 is the initial value
   const [orders, setOrders] = useState(0);
 
+  function OrderOne() {
```

Comment:
This is definitely an interesting and good use of a closure (a function that captures a variable from its surrounding scope, in this case the `setOrders` function). Code convention wise I'd expect the naming of a function to be camel-case, e.g. `orderOne`. (The functional components have PascalCase naming because they're React components and that's an overriding convention in the naming of a React component, but generally functions in JavaScript should have camel-case naming.)

Reply:
So for a React component can use PascalCase but functions used in a component file should use camelCase. I've changed OrderOne to orderOne now, thanks!